src/ifcwrap-csharp/IfcOpenShell.Net/IfcOpenShell.TestApp/Program.cs

```diff
@@ -0,0 +1,134 @@
+﻿// See https://aka.ms/new-console-template for more information
```

Comment:
Have a look at this commented-out code for the entire file, please.

Consider adding a comment why this exists / how to use or removing the file / test app completely